zyte_api/aio/retry.py

```diff
@@ -51,7 +51,7 @@ def _is_network_error(exc: BaseException) -> bool:
 
 
 def _is_throttling_error(exc: BaseException) -> bool:
-    return isinstance(exc, RequestError) and exc.status in (429, 503)
+    return isinstance(exc, RequestError) and exc.status in (429, 503, 520)
 
```

Comment:
For these errs, we'll need to follow the value specified in the `Retry-After` header.

What do you think about updating https://github.com/zytedata/python-zyte-api/blob/main/zyte_api/aio/retry.py#L71-L81 to check the headers?

Although we could do that in a separate PR to prevent blocking this quick fix.